docs/database/beta/getting-started.md

```diff
@@ -0,0 +1,53 @@
+# Getting started
+
+## Beta access
+
+As we are still in beta, we are slowly opening access, so ensure we can provide the best quality possible.
+
+To get into the database beta program, send us an Email to:
+
+`support@vapor.cloud`
+
+**Subject:** Database beta
+
+With the following information:
+
+- Email added in Vapor Cloud
+- What database engines you want to use
+- Number of database servers you will be starting
+
+## Create your database
+
+To create your new database, you can simply run.
+
+```
+vapor-beta cloud database create
+```
+
+This will create a full database server, and a database for each environment.
+
+When you create your database server, we will automatically setup a configuration variable on each of your environments called: `<TOKEN>` this will contain the connect URL to your database.
+
+In your project config file, you can simply read it with `$<TOKEN>`.
+
+Most database commands will require this name to be sent, for example:
+
+```
+vapor-beta cloud database inspect --token=<TOKEN>
+```
+
+### Multiple databases with same engine
+
+Let's say you want to have 2 database servers both PostgreSQL.
+
+The name you sent, will be added to the secondary database, so if i give it the name `test`
+
+The secondary will have the token `DB_MYSQL_TEST`
+
+You can always promote the secondary database to primary by running
+
+```
+vapor-beta cloud database promote DB_MYSQL_TEST
+```
+
+This will promote our secondary and demote our primary.
```

Comment:
what's the difference between a primary and secondary database?

Reply:
Whoops, removing that :D